modules/020-deckhouse/images/webhook-handler/werf.inc.yaml

```diff
@@ -12,7 +12,7 @@ ansible:
       unarchive:
         extra_opts:
           # The promtool version should match prometheus version from the 300-prometheus module.
-         {{- $promVersion := "2.36.2" }}
+         {{- $promVersion := "2.37.0" }}
```

Comment:
Title says `update. to 3.37.0` and here is `2.37.0`. Don't want to sync it?

Reply:
👍